app/controllers/expenses_additional_sources_controller.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class ExpensesAdditionalSourcesController < StandardStepsController
+  private
+
```

Comment:
@jessieay what's the rule for the `existing_attributes` method? Is the goal to provide a hash with the existing model's current state as it pertains to *this* action?  So for here I would be adding?

```ruby
  def existing_attributes
    HashWithIndifferentAccess.new(application_attributes)
  end

  def application_attributes
    app = current_snap_application

    {
      monthly_care_expenses: app.monthly_care_expenses,
      monthly_medical_expenses: app.monthly_medical_expenses,
      monthly_court_ordered_expenses: app.monthly_court_ordered_expenses,
      monthly_tax_deductible_expenses: app.monthly_tax_deductible_expenses,
      care_expenses: app.care_expenses,
      medical_expenses: app.medical_expenses,
      court_ordered_expenses: app.court_ordered_expenses,
      tax_deductible_expenses: app.tax_deductible_expenses,
    }
  end
```

Is that right?  And is that tested with that shared example?